src/helm/common/audio_utils.py

```diff
@@ -20,3 +20,20 @@ def ensure_wav_file_exists_from_array(path: str, array: np.ndarray, sample_rate:
         tmp_path = f"{path_prefix}.tmp.wav"
         wavfile.write(tmp_path, array, samplerate=sample_rate)
         os.rename(tmp_path, path)
+
+
+def ensure_mp3_file_exists_from_array(path: str, array: np.ndarray, sample_rate: int) -> None:
```

Comment:
As far as I can tell, this does not have the intended effect of writing an mp3 file - it simply writes a wav file to the given path.